deploy

```diff
@@ -2,21 +2,35 @@
 
 echo "
 -------------------------------------------------------------------------------
-ZK NFT Deployment Script
+NFT Deployment Script
 -------------------------------------------------------------------------------
 This script deploys an instance of a Centrifuge NFT 
 
 Prerequisites:
-* This script uses the Seth Ethereum CLI which can be installed from http://dapp.tools by running `curl https://dapp.tools/install | sh`
-* Set the variable ANCHOR_REPOSITORY to an instance of the Centrifuge Anchor Repository (https://github.com/centrifuge/centrifuge-ethereum-contracts/blob/develop/contracts/AnchorRepository.sol). 
-* If you want to use an existing deployment of the anchor registry, they can be found here: https://github.com/centrifuge/go-centrifuge/blob/develop/build/configs/default_config.yaml
+* This script uses the Seth Ethereum CLI which can be installed from http://dapp.tools by running 'curl https://dapp.tools/install | sh'
+* Set the variable ASSET_REPOSITORY to an instance of the Centrifuge Asset Repository (https://github.com/centrifuge/ethereum-bridge-contracts/blob/master/src/bridge_asset.sol).
+* If you want to use an existing deployment of the asset registry, it can be found here: https://github.com/centrifuge/go-centrifuge/blob/develop/build/configs/default_config.yaml
 
 "
 set -ex
 
-REGISTRY=$(seth send --create out/NFT.bin 'NFT(string memory, string memory, address)' "Centrifuge NFT" "NFT" $ANCHOR_REPOSITORY)
+export CONTRACT_NAME=${CONTRACT_NAME:-NFT}
+
+export CENT_ENV=${CENT_ENV:-"fulvous"}
+export ETH_GAS=${ETH_GAS:-4712388}
+export ETH_KEYSTORE=${ETH_KEYSTORE:-./keystore}
```

Comment:
These are default gas options that shouldn't be in there

Reply:
This is deployment script, do we need it @mikiquantum ?